webauthn4j-core/src/test/java/com/webauthn4j/data/attestation/authenticator/EC2CredentialPublicKeyTest.java

```diff
@@ -157,40 +157,49 @@ void validate_from_generated_key_01() throws Exception {
         KeyPairGenerator keyPairGenerator = KeyPairGenerator.getInstance("EC");
         keyPairGenerator.initialize(ECUtil.P_256_SPEC);
 
-        for (int i = 0; i < 1000; i++) {
-            KeyPair keyPair = keyPairGenerator.generateKeyPair();
-
-            PublicKey publicKey = keyPair.getPublic();
-            EC2CredentialPublicKey ec2CredentialPublicKey = TestDataUtil.createECCredentialPublicKey((ECPublicKey) publicKey);
-            ec2CredentialPublicKey.validate();
-        }
+        testMultipleValidationWithNewKeyPairs("P_256_SPEC", keyPairGenerator);
     }
 
     @Test
     void validate_from_generated_key_02() throws Exception {
         KeyPairGenerator keyPairGenerator = KeyPairGenerator.getInstance("EC");
         keyPairGenerator.initialize(ECUtil.P_384_SPEC);
 
-        for (int i = 0; i < 1000; i++) {
-            KeyPair keyPair = keyPairGenerator.generateKeyPair();
-
-            PublicKey publicKey = keyPair.getPublic();
-            EC2CredentialPublicKey ec2CredentialPublicKey = TestDataUtil.createECCredentialPublicKey((ECPublicKey) publicKey);
-            ec2CredentialPublicKey.validate();
-        }
+        testMultipleValidationWithNewKeyPairs("P_384_SPEC", keyPairGenerator);
     }
 
     @Test
     void validate_from_generated_key_03() throws Exception {
         KeyPairGenerator keyPairGenerator = KeyPairGenerator.getInstance("EC");
         keyPairGenerator.initialize(ECUtil.P_521_SPEC);
 
-        for (int i = 0; i < 1000; i++) {
+        testMultipleValidationWithNewKeyPairs("P_521_SPEC", keyPairGenerator);
+    }
+
+    private void testMultipleValidationWithNewKeyPairs(String name, KeyPairGenerator keyPairGenerator) {
+        for (int i = 0; i < 5; i++) {
             KeyPair keyPair = keyPairGenerator.generateKeyPair();
 
             PublicKey publicKey = keyPair.getPublic();
             EC2CredentialPublicKey ec2CredentialPublicKey = TestDataUtil.createECCredentialPublicKey((ECPublicKey) publicKey);
+
+            byte[] bytesOfX = ((ECPublicKey) publicKey).getW().getAffineX().toByteArray();
```

Comment:
As `((ECPublicKey) publicKey).getW().getAffineX().toByteArray()` returns two's-complement representation of the BigInteger, its length is not fixed.
The actual authenticator like Yubikey or Feitian BioPass puts fixed array representation to the x or y field, as far as I observed.

But... it is just an observation, and I couldn't find any spec that ensures the field is fixed array.
May be it's better to remove the length check.